cosmos/azure-spring-data-cosmos/cosmos-multi-tenant-samples/multi-tenant-by-container/src/main/java/com/azure/spring/data/cosmos/example/OrderRepository.java

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.spring.data.cosmos.example;
+
+import com.azure.spring.data.cosmos.repository.CosmosRepository;
+import com.azure.spring.data.cosmos.repository.Query;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+@Repository
+public interface OrderRepository extends CosmosRepository<Order, String> {
+    //get orders with custom query filtering by type since entities are co-located in same container
```

Comment:
It will be easier if we do not bring `co-located` in this sample.

Reply:
Can you expand? The practice of co-locating entities is important and useful for Cosmos DB. This is the first time we are introducing it for Spring Data, because it is the first time it has been feasible (and it is a by-product of multi-tenant support at container level) It is also a differentiating feature.

Reply:
@chenrujun - it is important to talk about co-location here, since that's how multi-tenancy will be achieved here.